core/input/input_event.cpp

```diff
@@ -1327,7 +1327,12 @@ bool InputEventAction::action_match(const Ref<InputEvent> &p_event, bool p_exact
 }
 
 String InputEventAction::as_text() const {
-	return vformat(RTR("Input Action %s was %s"), action, pressed ? "pressed" : "released");
+	const List<Ref<InputEvent>> *events = InputMap::get_singleton()->action_get_events(action);
+	if (!events->is_empty()) {
+		return events->front()->get()->as_text();
+	}
+
+	return TTR("No Events");
```

Comment:
`TTR` stands for "Tools TRanslate`, it shouldn't be used outside editor code.

But also here it shouldn't be used at all, we shouldn't localize these strings. I see `InputEventKey::to_string` uses `TTR("none")`, this should be changed too. I'll do a PR for this as I spot other localization issues in that code.

Reply:
Wouldn't we want things like Left Mouse Button, Right Mouse Button, etc translated? What about `RTR()`?

Reply:
Yeah the button names seem to use `TTRC` but then they don't properly use `TTRGET` to actually resolve the strings, so I guess their localization doesn't work. Editor translations also aren't included in the templates so those translations won't work at runtime unless the users provide the translation themselves.